Tests/TensorFlowTests/TensorAutoDiffTests.swift

```diff
@@ -1104,6 +1104,11 @@ final class TensorAutoDiffTests: XCTestCase {
       ])
   }
 
+  func testGelu() {
+    func f(a: Tensor<Float>) -> Tensor<Float> { gelu(a).sum() }
+    XCTAssertEqual(gradient(at: [5, -5, 0], in: f), [1, 0, 0.5])
```

Comment:
Use `XCTAssert(gradient(at: [5, -5, 0], in: f).isAlmostEqual(to: [1, 0, 0.5]))` instead, this doesn't pass on my machine.

Reply:
Done.  It matched exactly on my machine so I had to guess a tolerance level.